frontend/scss/app/components/_text_button.scss

```
@@ -31,6 +33,39 @@
+    transition:
+      background-color 160ms ease,
+      color 160ms ease,
+      transform 120ms ease;
+
+    &::after {
+      content: "";
+      position: absolute;
+      inset: 0;
+      pointer-events: none;
+      border-radius: inherit;
+      background-image: radial-gradient(circle, currentColor 10%, transparent 10.5%);
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)